inc/api.php

```diff
@@ -182,6 +183,39 @@ function stubEscapeFunctions()
             ]
         );
     }
+
+    /**
+     * Stub URL-related functions with default behavior.
+     */
+    function stubWpUrlFunctions($domain = 'example.org', $use_https = null)
+    {
+        $helper = new UrlsHelper($domain, $use_https);
+
+       stubs([
+            'home_url' => $helper->stubUrlCallback(),
+            'get_home_url' => $helper->stubUrlForSiteCallback(),
+            'site_url' => $helper->stubUrlCallback(),
+            'get_site_url' => $helper->stubUrlForSiteCallback(),
+            'admin_url' => $helper->stubUrlCallback('wp-admin', 'admin'),
+            'get_admin_url' => $helper->stubUrlForSiteCallback('wp-admin', 'admin'),
+            'content_url' => $helper->stubUrlCallback('wp-content', null, false),
+            'rest_url' => $helper->stubUrlCallback('wp-json'),
+            'get_rest_url' => $helper->stubUrlForSiteCallback('wp-json'),
+            'includes_url' => $helper->stubUrlCallback('wp-includes'),
+            'network_home_url' => $helper->stubUrlCallback(),
+            'network_site_url' => $helper->stubUrlCallback(),
+            'network_admin_url' => $helper->stubUrlCallback('wp-admin/network', 'admin'),
+            'user_admin_url' => $helper->stubUrlCallback('wp-admin/user', 'admin'),
+            'wp_login_url' => static function ($redirect = '', $force_reauth = false) use ($helper) {
+                $callback = $helper->stubUrlCallback();
+                $url = $callback('/wp-login.php', 'login');
```

Comment:
Small simplification as the `$callback` variable isn't used anywhere else within the closure.

```suggestion
                $url = $helper->stubUrlCallback('/wp-login.php', 'login');
```

Reply:
@jrfnl `stubUrlCallback()` return a *callback that returns a string*, not a string.
Following your suggestion `$url` would be a closure, not a string.
The simplification would be:  `$url = $helper->stubUrlCallback()('/wp-login.php', 'login');` but that does not work on PHP 5.6 where this lib is still stuck.

Reply:
Darn, of course!